Package.swift

```diff
@@ -23,15 +23,15 @@ let package = Package(
     .library(name: "SwiftGRPC", targets: ["SwiftGRPC"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/Zewo/zlib.git", from: "0.4.0"),
+    .package(url: "https://github.com/apple/swift-nio-zlib-support.git", from: "1.0.0"),
```

Comment:
I'm not hugely-confident that this project won't move, but since the zewo version was deprecated, I'm ok with this.